### ISSUE TITLE
Release 1.29.0 broken, AttributeError: 'InfluxDBClient' object has no attribute 'auth_header_value'

### ISSUE THREAD

**Issue by mmatl** [influxdata/influxdb-client-python#442]
__Steps to reproduce:__

Run the following python code:
```python
from influxdb_client import InfluxDBClient

x = InfluxDBClient("localhost:8086")
```

__Expected behavior:__
Not a crash.

__Actual behavior:__
Crash:

```
Traceback (most recent call last):
  File "/Users/mmatl/ambi/tmp.py", line 3, in <module>
    x = InfluxDBClient("localhost:8086")
  File "/Users/mmatl/ambi/env/lib/python3.9/site-packages/influxdb_client/client/influxdb_client.py", line 62, in __init__
    header_value=self.auth_header_value, retries=self.retries)
AttributeError: 'InfluxDBClient' object has no attribute 'auth_header_value'
Exception ignored in: <function InfluxDBClient.__del__ at 0x10e0efb80>
Traceback (most recent call last):
  File "/Users/mmatl/ambi/env/lib/python3.9/site-packages/influxdb_client/client/influxdb_client.py", line 264, in __del__
    if self.api_client:
AttributeError: 'InfluxDBClient' object has no attribute 'api_client'
```

__Specifications:__
 - Client Version: 1.29.0
 - InfluxDB Version:
 - Platform: MacOS, Ubuntu, Windows (all tested)

**Comment by bednar** [comment]
Hi @mmatl,

thanks for using our client.

We will release fixed version ASAP. As a workaround you can use following code:

```python
InfluxDBClient.auth_header_value = None
client = InfluxDBClient("http://localhost:8086")
```

Regards

**Comment by bednar** [comment]
Hi @mmatl,

I have prepared a fixed version within #443. If you would like to use this fixed version before regular release, please install client via:

```
pip install git+https://github.com/influxdata/influxdb-client-python.git@fix/fix_without_token
```

Regards
